modules/games.js

```diff
@@ -0,0 +1,326 @@
+        if (++i == 25) break;
+      }
+      if (games.length == 0) {
+        result = await chessAPI.getPlayerStats(encodeURIComponent(name));
+        if (result) {
+          const daily = result.body["chess_daily"];
+          const tactics = result.body["tactics"];
+          const puzzle = result.body["puzzle_rush"]?.best;
+          const toTime = (time) => `<t:${time}:F>`;
+          const overall = daily?.record ? `Overall:\nWins: ${daily.record.win}\nLosses: ${daily.record.loss}\nDraws: ${daily.record.draw}\nTime Per Move: ${daily.record['time_per_move']}\n\n` : "";
+          const latest = daily?.last ? `Latest:\nRating: ${daily.last.rating}\nDate: ${toTime(daily.last.date)}\n\n` : "";
+          const best = daily?.best ? `Best:\nRating: ${daily.best.rating}\nDate: ${toTime(daily.best.date)}\n[Link](${daily.best.game})` : "";
+          embed.setTitle(`Chess.com Stats for ${name}`)
+            .addField("Chess Daily", (overall || latest || best) ? `${overall}${latest}${best}` : "No available stats")
+            .addField("Puzzle Rush", puzzle ? `Total Attempts: ${puzzle["total_attempts"]}\nHigh Score: ${puzzle.score}` : "No available stats")
+            .addField("Tactics", tactics ? `Highest Rating: ${tactics.highest.rating}\nLowest Rating: ${tactics.lowest.rating}` : "No available stats");
+          return inter.reply({ embeds: [embed] });
+        }
+      }
+      if (games.length == 0) embed.setDescription(`No active games found for ${name}`);
+      if (games.length > 25) embed.setDescription(`${name}'s first 25 active games:`);
+      else embed.setDescription(`${name}'s active games:`);
+      inter.reply({ embeds: [embed] });
+    } catch (error) {
+      if (error.message == "Not Found" && error.statusCode == 404) {
+        inter.reply({ content: `I couldn't find a profile for \`${name}\`.`, ephemeral: true });
+      } else { u.errorHandler(error, inter); }
+    }
+  } else {
+    inter.reply({ content: "I couldn't find a saved IGN for them.", ephemeral: true });
+  }
+}
+
+async function eliteGetStatus() {
+  const status = await eliteAPI.getEliteStatus();
+  return { content: `The Elite: Dangerous servers are ${status.type == 'success' ? "online" : "offline"}`, ephemeral: true };
+}
+
+function eliteGetTime() {
+  const d = new Date();
+  const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+  return `The current date/time in Elite is ${monthNames[d.getUTCMonth()]} ${d.getUTCDate()}, ${(d.getUTCFullYear() + 1286)}, ${d.getUTCHours()}:${d.getUTCMinutes()}. (UTC + 1286 years)`;
+}
+
+/**
+ * @param {string} system
+ * @param {discord.MessageEmbed} embed
+ */
+async function eliteGetSystem(system, embed) {
+  if (!system) return { content: "I couldn't find a system with that name.", ephemeral: true };
```

Comment:
This can be done in `slashGameElite` instead of placing it in each function. (DRY)